### PR TITLE
Update apiDefinition.swagger.json

### PR DESCRIPTION
**Pull request by sjaishanker**
Modification in the description for SkyPoint Cloud

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/SkyPointCloud/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "SkyPoint Cloud",
-    "description": "SkyPoint is a customer data platform that champions privacy and compliance through connected privacy-first customer experiences. SkyPoint Connector provides a set of APIs to work with SkyPoint platform instances and entities.",
+    "description": "SkyPoint Cloud is the industry's first Modern Data Stack Platform with built-in data lakehouse, account 360, customer 360, identity resolution, data privacy vault, ELT / Reverse ETL, data integration, privacy compliance automation, data governance, analytics, partner integrations, and managed services for organizations in several industries including healthcare, life sciences, senior living, retail, hospitality, business services, and financial services. SkyPoint Connector provides a set of APIs to work with SkyPoint Modern Data Stack Platform solutions",
```
Comment (Amjed-Ayoub) [inline certified-connectors/SkyPointCloud/apiDefinition.swagger.json:5]: @sjaishanker The 'description' value must be between '30' and '500' characters. thank you so much

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/SkyPointCloud/apiDefinition.swagger.json (hunk 1 above)
Comment (sjaishanker) [reply certified-connectors/SkyPointCloud/apiDefinition.swagger.json]: Sure @Amjed-Ayoub, will make the changes. Thanks

**Comment by Amjed-Ayoub** [comment]
@sjaishanker you opened a new PR, please let us know if we should close this pull request?

**Comment by Amjed-Ayoub** [comment]
Closing this PR since the new one got approved and merged. 
[New PR](https://github.com/microsoft/PowerPlatformConnectors/pull/2195)

**Pull request closed without merging**